### DISCUSSION TITLE
7.8.5版本的安卓版lantern会导致手机卡死然后重启

### DISCUSSION THREAD

**Discussion by Lurker1011** [getlantern/lantern-forum-cn#454] [General]
最新版的安卓版lantern运行后会花一定时间链接节点，链接上节点之后没多久（这时才能正常进被墙的网址），手机就会卡死，大概10秒左右就重启，回滚到7.6.15版本则是正常运行的
手机是nova5 pro 系统是HamonyOS 4.00

**Comment by MaxMKMax** [comment]
@Lurker1011 您好，谢谢您的反馈。非常抱歉影响了您的体验感。

您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

1. go to Support (转到支持)
2. select Report an Issue (选择报告问题)
3. choose issue type (选择问题类型)
4. include additional details (包括额外的详细信息)
5. hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 不好意思, 需要您的帮忙. 谢谢!

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。

**Comment by MaxMKMax** [comment]
https://github.com/getlantern/lantern
同时您也可以下载最新版本. 8.0.0
